observability/tools/loki.md

```diff
@@ -0,0 +1,39 @@
+# Loki
+
+Loki is a horizontally-scalable, highly-available, multi-tenant log aggregation system, created by Grafana
+Labs inspired by the learnings from Prometheus. Loki is commonly referred as Prometheus but for logs, which
+makes total sense once both tools follow the same architecture, which is a agent collecting metrics in each
```

Comment:
Suggestion to break the second part of the sentence into a separate sentence for clarity

Reply:
Just did some changes for better clarity, what do you think?